src/uk/gov/hmcts/contino/GradleBuilder.groovy

```diff
@@ -38,6 +38,10 @@ class GradleBuilder extends AbstractBuilder {
       gradle("--info ${properties} sonarqube")
   }
 
+  def highLevelDataSetup(String dataSetupEnvironment) {
+    gradle("highLevelDataSetup --args=${dataSetupEnvironment}")
```

Comment:
don't you need secrets for importing? do you plan to handle this in the pipeline or in each script?

before merge we need a complete reference implementation so we can see what this looks like for a consumer

Reply:
We will need secrets, yes. We will have them on the adopting Data Store PR, as a reference example.

Reply:
We are working on this